cypress/integration/ampSpec.js

```diff
@@ -17,6 +17,12 @@ describe('AMP Tests', () => {
     getElement('html').should('have.attr', 'amp');
   });
 
+  it('should load the AMP framework', () => {
+    // .eq(1) gets the amp <script> as the first loaded is a Cypress <script>
+    const ampScript = getElement('html script').eq(1);
```

Comment:
It would be good to update this cypress test to be more specific - to check this script is in the head of the page. e.g. with this selector: `'head script'`